libbeat/cmd/instance/imports_linux_darwin_windows.go

```diff
@@ -0,0 +1,27 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
We could maybe name this file `imports_docker.go`, so we know that they are imports related to docker, and we don't need to rename the file if list of supported OSs change.